loadtest/config.go

```diff
@@ -24,20 +24,20 @@ type ConnectionConfiguration struct {
 }
 
 // IsValid reports whether a given ConnectionConfiguration is valid or not.
```

Comment:
Here and in a few cases below I think we should make the doc string consistent and add `Returns an error if the validation fails.`